### ISSUE TITLE
firebase_app_distribution issue

### ISSUE THREAD

**Issue by nachkar** [fastlane/fastlane#20059]
I got this error each time I try to submit my iPA to Firebase App Distribution service : 

You can ⌘ + double-click on links to open them directly in your browser.
/Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/response/raise_error.rb:22:in `on_complete': [!] the server responded with status 404 (Faraday::ResourceNotFound)
	from /Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/middleware.rb:19:in `block in call'
	from /Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/response.rb:61:in `on_complete'
	from /Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/middleware.rb:18:in `call'
	from /Library/Ruby/Gems/2.6.0/gems/faraday_middleware-1.2.0/lib/faraday_middleware/response_middleware.rb:36:in `call'
	from /Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/rack_builder.rb:154:in `build_response'
	from /Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/connection.rb:516:in `run_request'
	from /Library/Ruby/Gems/2.6.0/gems/faraday-1.10.0/lib/faraday/connection.rb:281:in `post'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-plugin-firebase_app_distribution-0.3.2/lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb:112:in `upload_binary'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-plugin-firebase_app_distribution-0.3.2/lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb:141:in `upload'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-plugin-firebase_app_distribution-0.3.2/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:41:in `run'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:263:in `block (2 levels) in execute_action'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/actions/actions_helper.rb:69:in `execute_action'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:255:in `block in execute_action'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:229:in `chdir'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:229:in `execute_action'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
	from Fastfile:58:in `block (2 levels) in parsing_binding'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/commands_generator.rb:109:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:124:in `run!'
	from /Library/Ruby/Gems/2.6.0/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/commands_generator.rb:353:in `run'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/fastlane/lib/fastlane/cli_tools_distributor.rb:122:in `take_off'
	from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.204.3/bin/fastlane:23:in `<top (required)>'
	from /usr/local/bin/fastlane:23:in `load'
	from /usr/local/bin/fastlane:23:in `<main>'

Fastlane Env :
<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 12.2.1                                                         |
| Ruby                        | 2.6.8                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.35.1                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 12.2.1 (21D62)                                           |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 13.2.1                                                         |
| Swift Version               | 5.5.2                                                          |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

<details><summary>`./fastlane/Appfile`</summary>

```ruby
# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
# apple_id("[[APPLE_ID]]") # Your Apple email address


# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.204.3 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                    | Version | Update-Status |
| ----------------------------------------- | ------- | ------------- |
| fastlane-plugin-firebase_app_distribution | 0.3.2   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                       | Version      |
| ----------------------------------------- | ------------ |
| did_you_mean                              | 1.3.0        |
| atomos                                    | 0.1.3        |
| colored2                                  | 3.1.2        |
| nanaimo                                   | 0.3.0        |
| rexml                                     | 3.2.5        |
| xcodeproj                                 | 1.21.0       |
| rouge                                     | 2.0.7        |
| xcpretty                                  | 0.3.0        |
| terminal-notifier                         | 2.0.0        |
| unicode-display_width                     | 1.8.0        |
| terminal-table                            | 1.8.0        |
| plist                                     | 3.6.0        |
| public_suffix                             | 4.0.6        |
| addressable                               | 2.8.0        |
| multipart-post                            | 2.0.0        |
| word_wrap                                 | 1.0.0        |
| optparse                                  | 0.1.1        |
| tty-screen                                | 0.8.1        |
| tty-cursor                                | 0.7.1        |
| tty-spinner                               | 0.9.3        |
| artifactory                               | 3.0.15       |
| babosa                                    | 1.0.4        |
| colored                                   | 1.2          |
| highline                                  | 2.0.3        |
| commander                                 | 4.6.0        |
| unf                                       | 0.1.4        |
| domain_name                               | 0.5.20190701 |
| http-cookie                               | 1.0.4        |
| faraday-cookie_jar                        | 0.0.7        |
| faraday_middleware                        | 1.2.0        |
| gh_inspector                              | 1.1.3        |
| mini_magick                               | 4.11.0       |
| naturally                                 | 2.2.1        |
| rubyzip                                   | 2.3.2        |
| security                                  | 0.1.3        |
| xcpretty-travis-formatter                 | 1.0.1        |
| dotenv                                    | 2.7.6        |
| bundler                                   | 2.1.4        |
| simctl                                    | 1.6.8        |
| jwt                                       | 2.3.0        |
| emoji_regex                               | 3.2.3        |
| CFPropertyList                            | 3.0.5        |
| claide                                    | 1.1.0        |
| excon                                     | 0.92.0       |
| ruby2_keywords                            | 0.0.5        |
| faraday-retry                             | 1.0.3        |
| faraday-rack                              | 1.0.0        |
| faraday-patron                            | 1.0.0        |
| faraday-net_http_persistent               | 1.2.0        |
| faraday-net_http                          | 1.0.1        |
| faraday-multipart                         | 1.0.3        |
| faraday-httpclient                        | 1.0.1        |
| faraday-excon                             | 1.1.0        |
| faraday-em_synchrony                      | 1.0.0        |
| faraday-em_http                           | 1.0.0        |
| faraday                                   | 1.10.0       |
| unf_ext                                   | 0.0.8.1      |
| fastimage                                 | 2.2.6        |
| json                                      | 2.6.1        |
| webrick                                   | 1.7.0        |
| httpclient                                | 2.8.3        |
| multi_json                                | 1.15.0       |
| signet                                    | 0.16.1       |
| os                                        | 1.1.4        |
| memoist                                   | 0.16.2       |
| googleauth                                | 1.1.2        |
| mini_mime                                 | 1.1.2        |
| retriable                                 | 3.1.2        |
| trailblazer-option                        | 0.1.2        |
| declarative                               | 0.0.20       |
| uber                                      | 0.1.0        |
| representable                             | 3.1.1        |
| google-apis-core                          | 0.4.2        |
| google-apis-playcustomapp_v1              | 0.7.0        |
| google-apis-androidpublisher_v3           | 0.16.0       |
| rake                                      | 13.0.6       |
| digest-crc                                | 0.6.4        |
| google-apis-storage_v1                    | 0.11.0       |
| google-apis-iamcredentials_v1             | 0.10.0       |
| google-cloud-errors                       | 1.2.0        |
| google-cloud-env                          | 1.5.0        |
| google-cloud-core                         | 1.6.0        |
| google-cloud-storage                      | 1.36.1       |
| aws-eventstream                           | 1.2.0        |
| aws-sigv4                                 | 1.4.0        |
| aws-partitions                            | 1.566.0      |
| jmespath                                  | 1.6.1        |
| aws-sdk-core                              | 3.130.0      |
| aws-sdk-kms                               | 1.55.0       |
| aws-sdk-s3                                | 1.113.0      |
| forwardable                               | 1.2.0        |
| logger                                    | 1.3.0        |
| date                                      | 2.0.0        |
| stringio                                  | 0.0.2        |
| ipaddr                                    | 1.2.2        |
| openssl                                   | 2.1.2        |
| zlib                                      | 1.0.0        |
| mutex_m                                   | 0.1.0        |
| ostruct                                   | 0.1.0        |
| strscan                                   | 1.0.0        |
| io-console                                | 0.4.7        |
| fileutils                                 | 1.1.0        |
| etc                                       | 1.0.1        |
| libxml-ruby                               | 3.2.1        |
| psych                                     | 3.1.0        |
| fastlane-plugin-firebase_app_distribution | 0.3.2        |
</details>

*generated on:* **2022-03-14**
</details>

**Comment by nachkar** [comment]
> It seems like you have not included the output of `fastlane env`
> To make it easier for us help you resolve this issue, please update the issue to include the output of `fastlane env` 👍

The required informations were added to the previous comment.

**Comment by rjuevesano** [comment]
@nachkar were you able to fix this?

**Comment by nachkar** [comment]
> @nachkar were you able to fix this?

No still not fixed couldnt find a solution.

**Comment by rjuevesano** [comment]
> > @nachkar were you able to fix this?
> 
> No still not fixed couldnt find a solution.

I was able to fixed by updating my bundler to `2.3.10` and updating firebase app distribution to the latest version.

**Comment by nachkar** [comment]
> > > @nachkar were you able to fix this?
> > 
> > No still not fixed couldnt find a solution.
> 
> I fixed it by just updating my bundler to `2.3.10` and updating firebase app distribution to the latest version.

It worked now thank you for your help.
